packages/create-styles/src/createCompiler/plugins/rtl.js

```diff
@@ -4,11 +4,12 @@
  */
 
 import cssjanus from 'cssjanus';
+import rtlcss from 'rtlcss';
 
-let isRtl = false;
-if (typeof window !== 'undefined') {
-	isRtl = window?.document?.documentElement?.dir === 'rtl';
-}
+let isRtl = true;
+// if (typeof window !== 'undefined') {
+// 	isRtl = window?.document?.documentElement?.dir === 'rtl';
+// }
```

Comment:
Temporary to make testing easier. You still have to add `dir="rtl"` to the `html` element in storybook.

Reply:
Awesome :). This definitely made it easier.

Feel free to revert this when you're ready 🎉